.github/workflows/test.js.yml

```diff
@@ -0,0 +1,54 @@
+name: Tests
+
+on:
+  push:
+    branches:
+      - develop
+  pull_request:
+    branches:
+      - develop
+
+jobs:
+  setup:
+    runs-on: ubuntu-latest
+    name: Compile and test MathJax
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - uses: pnpm/action-setup@v4
+        name: Install pnpm
+        with:
+          version: 10
+          run_install: false
+
+      - name: Install packages
+        run: |
+          pnpm -r i
+          pnpm link:src
+
+      - name: Temporarily make link
+        run: |
+          cd node_modules
+          ln -s @mathjax/src mathjax-full
+          cd ..
+          
+      - name: Compile MathJax
+        run: |
+          pnpm mjs:compile
+          components/bin/makeAll --mjs --terse --build components/mjs
+          pnpm cjs:compile
+          pnpm cjs:components:src:build
+          components/bin/makeAll --cjs --terse --build components/cjs
+          pnpm copy:pkg cjs
```

Comment:
Would you be willing to use `pnpm -s` rather than just `pnpm` for these (and the ones below)?  That way, you only see that script output rather than the pnpm commands themselves.  I think that looks cleaner.

Reply:
Done.